src/components/ResourceList/tests/ResourceList.test.tsx

```diff
@@ -750,6 +753,118 @@ describe('<ResourceList />', () => {
       resourceList.update();
       expect(resourceList.find(BulkActions).prop('selectMode')).toBe(false);
     });
+
+    describe('focus', () => {
+      describe('large screen', () => {
+        it('focuses the checkbox in the bulk action when the plain CheckableButton is clicked', () => {
+          const resourceList = mountWithAppProvider(
+            <ResourceList
+              items={itemsWithID}
+              renderItem={renderItem}
+              promotedBulkActions={promotedBulkActions}
+            />,
+          );
+
+          const selectAllCheckableButton = resourceList.findWhere(
+            (wrap) => wrap.is(CheckableButton) && wrap.prop('plain') === true,
```

Comment:
No strong feelings, we usually coerce these types as values rather than compare against a boolean but I'll leave the choice up to you 

```suggestion
            (wrap) => wrap.is(CheckableButton) && wrap.prop('plain'),
```